resources/setups/qualification/system_module_config.json

```diff
@@ -55,7 +55,7 @@
       "required": "pass",
       "expected": "Required Pass"
     },
-    "connection.switch.port_speed": {
+    "connection.switch.port_speedd": {
```

Comment:
speedd?  Extra d?

Reply:
Oh no! Awful typo! Thanks for pointing it out!